shipitscript/docker.d/init_worker.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
-set -e
+set -o errexit -o pipefail
+
+test_var_set() {
```

Comment:
Hm, should we define this once in `init.sh` and use that? I'm under the impression init.sh calls init_worker.sh, so we have access to the functions in both files.

Reply:
i'm not sure, but i will check